app-sso/troubleshoot.hbs.md

```diff
@@ -24,11 +24,35 @@ with:
 stern --all-namespaces --selector=app.kubernetes.io/managed-by=sso.apps.tanzu.vmware.com
 ```
 
-## Change propagation
+## Wait for change propagation
 
 When applying changes to an `AuthServer`, keep in mind that changes to issuer URI, IDP, server and logging configuration
 take a moment to be effective as the operator will roll out the authorization server `Deployment`.
 
+## Set up debug logs
+
+> **Caution** debug logs are very verbose, and display all identity information coming from usptream
+> identityProviders, e.g. all LDAP attributes for a given user. WMware recommends not using debug
+> logs in production.
```

Comment:
> WMware recommends not using debug logs in production.
**[issue]**: **V**Mware
**[nit]**: reword to _VMware does not recommend using debug logs in production._

Reply:
We actually recommend **not** to use debug logs, right? Maybe "recommends against" then?